docs/build-html.javascript

```diff
@@ -0,0 +1,55 @@
+/*
+ * build-html.javascript
+ * This program converts files in /docs/*.md to /out/docs/*.html
+ *
+ * This program-file has extension .javascript instead of .js
```

Comment:
perhaps it would be better to move it out of docs, rather than to use a non standard extension for it?

Additionally i doubt its being linted currently due to the different extension, and unlinted code shouldn’t land.

Reply:
renamed file /docs/build-html.javascript to /build-docs.js in commit 54231a55.

verified linting by inserting arbitrary syntax-error and failing the shell-command `npm run lint`.